Zend/zend_API.c

```diff
@@ -140,6 +140,22 @@ ZEND_API const char *zend_zval_type_name(const zval *arg) /* {{{ */
 }
 /* }}} */
 
+ZEND_API const char *zend_zval_type_error_name(const zval *arg) /* {{{ */
```

Comment:
I wasn't exactly sure that I can use `zend_zval_type_name()` for this purpose (although it seems likely for me) so I introduced this one.

Reply:
It seems that `phpdbg_info.c` remained the only client of `zend_zval_type_name()` about which I'm not sure if I could use the new semantics.

Reply:
I'm okay with reusing zend_zval_type_name for this and switching to that phpdbg use to zend_get_type_by_const.